Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
What about Behavioral.Automation.csproj ?

Reply:
Updated

Reply:
Not required for this project.